### PR TITLE
data:Lakorthus

### PR DESCRIPTION
**Pull request by Lakorthus**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5131"></a>

### CODE REVIEW

File: data/Lakorthus.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+  "name": "Julio Velezmoro",
+  "type": "personal",
+  "bio": "Open Source | MERN | Frontend Developer",
+  "avatar": "https://github.com/Lakorthus.png",
```
Comment (Ananya2001-an) [inline data/Lakorthus.json:5]: Please remove avatar key. Due to recent upgrades it's not needed anymore.
```suggestion

```

**Review by Ananya2001-an — APPROVED** [review]
Looks good! Thanks for making the changes.

**Comment by Lakorthus** [comment]
Thank you 🫶 for you assistance @Ananya2001-an

**Comment by Ananya2001-an** [comment]
> Thank you 🫶 for you assistance @Ananya2001-an 

No worries 😉

**Review by SurajPratap10 — APPROVED** [review]
LGTM

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**